.stylelintrc.js

```diff
@@ -86,14 +111,21 @@ module.exports = {
             }
         ],
         'font-family-name-quotes': 'always-unless-keyword',
-        // Turn off rules 
+        // Turn off rules
         'selector-class-pattern': null,
+        'selector-id-pattern': null,
```

Comment:
We can't change what the IDs look like at this point, it's too likely to cause problems.